app/controllers/public_pages_controller.rb

```diff
@@ -55,6 +55,16 @@ def sms_terms; end
 
   def diy; end
 
+  def cdss
```

Comment:
For whoever is reviewing, we were not sure this was the right protocol for adding a source. Do we want to save DIY sources separately from homepage sources?

Reply:
Setting the source directly like this is a little strange to me. I think it would make more sense to _not_ add these new routes, but to rather detect the source param in "home" and redirect to "diy" if source param is "cdss" or "ca"

Reply:
@bytheway875 good idea! How does this change look now? Is it okay that we added it to set_source?